tests/dummy/config/targets.js

```diff
@@ -1,4 +1,9 @@
 /* eslint-env node */
 module.exports = {
-  browsers: ['last 1 Chrome versions', 'last 1 Firefox versions', 'last 1 Safari versions'],
+  browsers: [
+    'ie 11',
```

Comment:
Lets do this conditionally? I really like debugging with native async/await, and we only need IE11 support in smaller cases.

I was thinking of something like: http://rwjblue.com/2017/10/30/async-await-configuration-adventure/#10limittargetsduringlocaldevelopmenttesting